qutebrowser/misc/cmdhistory.py

```diff
@@ -90,7 +90,7 @@ def start(self, text):
         else:
             items = self.history
         if not items:
-            raise HistoryEmptyError
+            raise HistoryEmptyError("History is empty.")
```

Comment:
Since those exceptions are only used programatically, I don't think they need to have a text - just leave the text off and only check the exception type:

```python
with pytest.raises(HistoryEmptyError):
    self.hist.start('k')
    # ...
```